test/decl/class/actor/conformance.swift

```diff
@@ -2,6 +2,8 @@
 
 // REQUIRES: concurrency
 
+import Distributed
```

Comment:
Now that we moved distributed test to another file, we can revert this import

Reply:
good catch - done.